source/less/blocks/additional-goods.less

```
@@ -0,0 +1,75 @@
+.additional-goods-title__wrapper{
```

```suggestion
.additional-goods__title-wrapper{
```

Так как это элемент БЭМ блока `additional-goods`. А ваше наименование класса говорит о том, что это элемент блока `additional-goods-title`.

---

Выполнил